Server/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp

```diff
@@ -11400,6 +11400,22 @@ bool CStaticFunctionDefinitions::GetAccountSerial(CAccount* pAccount, SString& s
     return bRegistered;
 }
 
+
+bool CStaticFunctionDefinitions::SetAccountSerial(SString strName, SString strSerial)
+{
+    const char* szAccountName = strName.c_str();
+    CAccount* pAccount = g_pGame->GetAccountManager()->Get(szAccountName);
+    
+    if (!pAccount)
+        return false;
+
+    bool bRegistered = pAccount->IsRegistered();
+    if (bRegistered)
+        pAccount->SetAccountSerial(strSerial);
+
+    return bRegistered;
```

Comment:
Avoid using Hungarian notation in new code

Reply:
Thanks for the feedback, I will avoid using future contributions <3